Client/src/App.css

```diff
@@ -0,0 +1,217 @@
+@import url('https://fonts.googleapis.com/css?family=Comfortaa|Gloria+Hallelujah|Lato|Mali|Zilla+Slab');
+
+
+
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 40vmin;
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+.myLogo{
+margin-top: 20px;
+width: 100%
+}
+
+.main-text{
+
+  font-size: 15px;
+  font-weight: bold;
+  font-family: Comfortaa;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin-top: 50px;
+
+}
+
+.main-text2{
```

Comment:
You should try to name your classes semantically for maintenance.  It's going to be hard to keep track of what these are in the future.